### DISCUSSION TITLE
dp_frz

### DISCUSSION THREAD

**Discussion by k50112113** [deepmodeling/deepmd-kit#110] [Q&A] [answered]
I encountered this error when I tried to freeze my trained model:

from ._conv import register_converters as _register_converters
Traceback (most recent call last):
  File "/home/scl6/project-beckman/deepmd-kit_r0.11/build/bin/dp_frz", line 81, in <module>
    freeze_graph(args.folder, args.output, args.nodes)
  File "/home/scl6/project-beckman/deepmd-kit_r0.11/build/bin/dp_frz", line 30, in freeze_graph
    input_checkpoint = checkpoint.model_checkpoint_path
AttributeError: 'NoneType' object has no attribute 'model_checkpoint_path'

I didn't face any error when I was training my model, so my question is:
1. It seems like a tensorflow problem, does this mean that I didn't install my tensorflow correctly?
2. Does dp_frz depends on tensorflow of python interface only?

**Comment by njzjz** [comment, marked as answer]
It seems that `tensorflow` cannot find your checkpoint. For the second question, yes.

**Reply by kaijiz** [reply]
could anyone tell me how to fix this problem? what is the meaning of checkpoint?

**Reply by njzjz** [reply]
A file named `checkpoint`. If it doesn't exist, something went wrong with `dp train`.
